src/Post_Type_Command.php

```diff
@@ -1,6 +1,8 @@
 <?php
 /**
- * Retrieves details on the site's registered post-types.
+ * Retrieves details on the site's registered post types.
+ *
+ * Get information on the WordPress' built-in and the site's [custom post types](https://developer.wordpress.org/plugins/post-types/).
```

Comment:
The first `the` (in front of `WordPress'`) seems grammatically wrong to me.

Reply:
Sorry, that was a mistake. Correcting now.

Also, I don't know what this means:
> please re-run wp scaffold package-readme to update the README.md file again.

Is that something I need to do on my end? If so, please send details.

Reply:
The `README.md` files inside of the command packages are automatically generated through the `wp scaffold package-readme` command: https://github.com/wp-cli/scaffold-package-command#wp-scaffold-package-readme

So, any time you change the documentation part of the source files, you'll need to re-run `wp scaffold package-readme . --force` from inside the root folder of that package to update the `README.md` file to the latest changes.

You might need to install the command first, if you haven't done so: `wp package install wp-cli/scaffold-package-command`

Reply:
Ah, I see: But I'm editing files in my forked repo directly on GitHub, not in a local version on my computer. So I'm guessing there is no way to run that package here on GitHub. Is there another way to update the README without me having to download the repo?

Reply:
We'll run the script over all packages once we've merged the updated descriptions.

Reply:
Thanks, @schlessera , that will save me some time (tho cost you some). Appreciated. We got diverted but have a few more PRs to submit for command descriptions. Will try to get to that today. And thanks to @miya0001 for all his help too.
cc @ako80218